appveyor.yml

```diff
@@ -6,7 +6,6 @@ platform:
 
 environment:
   matrix:
-    - ruby_version: "200-x64"
```

Comment:
Can you actually change this to 21-x64 so that we're prepared for first-class 64-bit client support.